csmaps_utils.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+#
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o nounset
+
+OC=oc
+YQ=yq
+CM_NAME="mock-cs-maps"
```

Comment:
assuming this should be changed back to common-service-maps

Reply:
I am leaning towards keeping it as `mock-cs-maps` since this file is primarily for testing and storing the functions to manipulate configmaps, not actually to be used as-is. If this script is included by another script, the CM_NAME can be overwritten by the importing script anyways